kevm-pyk/src/kontrol/__main__.py

```diff
@@ -468,15 +494,15 @@ def parse(s: str) -> list[T]:
     )
     foundry_prove_args.add_argument(
         '--test',
-        type=str,
+        type=_test_type,
         dest='tests',
```

Comment:
in the context of `foundry-prove`, we need to pass the tests with multiple `--test` flags, this will be in the form of `--test test,id` and can also be passed as `--test test` for having `id` to `None`. We need it for `foundry-prove` because `id` has to be grouped with `test`